algorithm/model.go

```diff
@@ -2,34 +2,43 @@ package algorithm
 
 import (
 	"context"
+	"fmt"
 	"time"
 
 	"github.com/go-redis/redis/v8"
 )
 
+var (
+	Registry []*RegInfo
+)
+
 type (
 	Limit interface {
-		GetAlgorithm() string
-		GetBurst() int64
 		GetRate() int64
+		GetBurst() int64
+		GetAlgorithm() string
 		GetPeriod() time.Duration
 	}
 
 	Rediser interface {
```

Comment:
what happened with the sorting here?

Reply:
i love how it looks from short to long ;)